Kotlin/graphql-simple-bindings-emitter/src/main/kotlin/community/flock/graphqlsimplebindings/emitter/KotlinEmitter.kt

```diff
@@ -12,10 +12,10 @@ class KotlinEmitter(private val packageName: String = "community.flock.graphqlsi
     override fun emitDocument(document: Document): String = super.emitDocument(document)
             .let { "package $packageName\n\n$it" }
 
-    override fun ObjectTypeDefinition.emitObjectTypeDefinition() = "data class $name(\n${fieldDefinitions.emitFields()}\n)\n"
+    override fun ObjectTypeDefinition.emitObjectTypeDefinition() = "data class $name(\n${fieldDefinitions.emitDefinitionFields()}\n)\n"
 
-    override fun List<FieldDefinition>.emitFields() = joinToString(",\n") { it.emitField() }
-    override fun FieldDefinition.emitField() = "        val $name: ${type.emitType()}"
+    override fun List<FieldDefinition>.emitDefinitionFields() = joinToString(",\n") { it.emitDefinitionField() }
+    override fun FieldDefinition.emitDefinitionField() = "\tval $name: ${type.emitType()}"
```

Comment:
Spaces over tabs obviously ;)

Reply:
Standard Intellij Kotlin formatter takes 8 spaces for a data class indented field.